addons/account/static/src/components/char_with_placeholder_field/char_with_placeholder_field.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="account.CharWithPlaceholderField" t-inherit="web.CharField">
+        <xpath expr="//span" position="attributes">
+            <attribute name="t-att-class">{'text-muted': !this.props.record.data[props.name]}</attribute>
+        </xpath>
```

Comment:
Isn't the field empty?
```suggestion
```

Reply:
In list view, when the user hasn't clicked on the element to edit it, it should show the placeholder value.
(There isn't any `input` in that case, so there's no way to implement the placeholder using the usual `placeholder=...`.)